src/test/java/nextstep/session/domain/DurationTest.java

```diff
@@ -0,0 +1,84 @@
+package nextstep.session.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class DurationTest {
+
+    @DisplayName("시작일자가 종료일자보다 미래면 IllegalArgumentException을 던진다.")
```

Comment:
테스트명을 작성할 때 구현체를 직접 네이밍 하는것 보다

```suggestion
    @DisplayName("시작일자가 종료일자보다 미래일 수 없다.")
```

위와 같이 작성하는건 어떨까요?
개인적으로 테스트코드는 비개발자도 읽을 수 있어야한다고 생각합니다 🤔 (문서의 역할)

Reply:
너무 멋진 관점이라고 생각합니다.
테스트 코드는 문서로서의 기능을 한다고 공부했는데, 예외와 같은 특정 명칭들이 있으면 어렵겠네요.

다른 부분들도 수정 했습니다!